lib/open_api_spex/plug/put_api_spec.ex

```diff
@@ -11,15 +11,12 @@ defmodule OpenApiSpex.Plug.PutApiSpec do
   @behaviour Plug
 
   @impl Plug
-  def init(opts = [module: _mod]), do: opts
+  def init([module: spec_module]), do: spec_module
```

Comment:
I don't know if this change is necessary, but I personally like it 🤷‍♂️

Reply:
Keyword lists aren't good for pattern matching. This will only match if there is only one item in the list.